Containerfile

```diff
@@ -85,21 +85,38 @@ RUN KERNEL_VERSION="$(rpm -q kernel --queryformat '%{VERSION}-%{RELEASE}.%{ARCH}
         rpm-ostree install \
             https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm \
             https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm \
-    && \
-        sed -i 's@enabled=1@enabled=0@g' /etc/yum.repos.d/rpmfusion-free{,-updates}.repo \
     && \
         rpm-ostree install \
             xorg-x11-drv-${NVIDIA_PACKAGE_NAME}-{,cuda-,devel-,kmodsrc-,power-}${NVIDIA_FULL_VERSION} \
             kernel-devel-${KERNEL_VERSION} nvidia-container-toolkit \
             "/tmp/akmods/${NVIDIA_PACKAGE_NAME}/kmod-${NVIDIA_PACKAGE_NAME}-${KERNEL_VERSION}-${NVIDIA_FULL_VERSION#*:}.rpm" \
             /tmp/ublue-os-nvidia-addons/rpmbuild/RPMS/noarch/ublue-os-nvidia-addons-*.rpm \
+    && \
+        rpm-ostree override remove $(rpm -qa --queryformat='%{NAME} ' \
+            mesa-va-drivers \
+            libavutil-free \
+            libswscale-free \
+            libswresample-free \
+            libavformat-free \
+            libavcodec-free \
+            libavfilter-free \
+            libavdevice-free \
+            libpostproc-free) \
+            --install=mesa-va-drivers-freeworld \
+            --install=mesa-vdpau-drivers-freeworld \
+            --install=libva-intel-driver \
+            --install=nvidia-vaapi-driver \
+            --install=ffmpeg-libs \
+            --install=ffmpeg \
+            --install=libavcodec-freeworld \
```

Comment:
Interesting choice here. I'm not sure you need this if you have `ffmpeg` installed via rpmfusion. For me I have `ffmpeg-free` from fedora along with `libavcodec-freeworld` from rpmfusion. This setup still seems to allow hardware accelerated video encode/decode. It prevents needing to override all the other `-free` packages locally.

But since this is a pre-packaged container, it seems sane to uninstall all the `-free` packages and install `ffmpeg`. Are you sure you still need `libavcodec-freeword`?

Also, do we want all the ffmpeg packages in this container specifically? Or the intel/AMD VAAPI drivers? (Intel is probably good for hybrid laptops)

I should mention that I'm no codec/ffmpeg expert at all. I'm having trouble finding any documentation on rpmfusion about all of this which leaves me very confused.

Reply:
I'm just thinking out loud. Feel free to disregard this if needed.

Reply:
ffmpeg-free is very cut down and ffmpeg is needed for Firefox to have full accelerated video. We can remove ffmpeg if we commit to only supporting flatpak Firefox.

Reply:
Ah, "this" was referring to `libavcodec-freeworld` specifically. I didn't seem to choose the option to highlight that single line. I'm just making sure we need `libavcodec-freeworld` if we have the fully featured `ffmpeg` since `libavcodec-freeworld` provides the patented codecs for usage with `ffmpeg-free`, but again I'm having trouble finding docs on it.